.github/workflows/push-build.yaml

```diff
@@ -0,0 +1,80 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+jobs:
+  push-master-image-on-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Prepare
+        run: |
+          DOCKER_IMAGE=${{ secrets.DOCKER_USERNAME }}/${GITHUB_REPOSITORY#*/}
```

Comment:
Let's write bash scripts instead

Reply:
Actually, we already have makefile commands, can we just use those?